.eslintrc

```diff
@@ -0,0 +1,42 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "plugin:prettier/recommended"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "globalThis": "readonly",
+    "SharedArrayBuffer": "readonly",
+    "PORT": "readonly",
+    "SESSION_SECRET": "readonly"
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "plugins": ["react", "json", "prettier"],
+  "rules": {
+    "indent": [2, 2, { "SwitchCase": 1 }],
+    "linebreak-style": ["error", "unix"],
+    "semi": ["error", "always"],
+    "eol-last": ["error", "always"],
+    "no-console": ["error", { "allow": ["warn", "error"] }],
+    "no-use-before-define": ["error"],
+    "react/display-name": ["off"]
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  }
+}
```

Comment:
This is certainly welcome, but I generally expect such changes to be done in dedicated pull requests. I also don't want to add extra work to your plate, so maybe we can just keep it here and any further "developer support" changes can be done in dedicated PRs.

Reply:
I didn't mean this to be a part of this PR! I'll revert it and put it in its own branch. Thanks!